releasenotes/2.0.0.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: releasenotes
+title: Release Notes for Version 2.0.0
+date: 2016-11-15 12:00:00
+release_notes: true
+permalink: /releasenotes/2.0.0/
+---
+
+1. [Introduction](#intro)
+1. [Release Highlights](#highlights)
+1. [Compute Improvements](#computeimprovements)
+1. [BlobStore Improvements](#blobstoreimprovements)
+1. [Known Issues](#knownissues)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+Apache jclouds 2.0 represents a significant milestone for the project. We are proud to support all major cloud providers in the marketplace with a mature, stable, codebase that is ready for production. You can [read the official announcement here](#). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
+
+## <a id="highlights"></a>Release Highlights
+
+Apache jclouds 2.0 features include:
+
+* Wider compatibility with the Guava and Guice libraries.
+* Configuration of arbitrary hardware values in the compute abstraction.
+* Added support for new cloud providers:
+  * Microsoft Azure Resource Manager
+  * ProfitBricks v3
+  * OneAndOne
+  * Backblaze B2
+* Better integration with OSGi and Apache Karaf.
+* Numerous bug fixes and performance improvements.
+
+The complete list of fixed issues and improvements can be found [here](https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12327379&styleName=Html&projectId=12314430).
+
+We have also improved our CI system by adding compatibility builds with the core libraries and by adding weekly builds that run the live tests. Check it out at: [https://jclouds.ci.cloudbees.com](https://jclouds.ci.cloudbees.com).
+
+## <a id="computeimprovements"></a>Compute Improvements
+
+* Added support for Microsoft azure ARM, ProfitBricks v3 and OneAndOne.
+* Configuration of arbitrary hardware values in the compute abstraction. [Read more](/blog/2016/08/22/arbitrary-cpu-ram/) aboout this useful feature fruit of a GSoC 2016 project!
```

Comment:
A feature fruit a day keeps the doctor away! ;-)

See comment above about the "arbitrary hardware values" phrase